docs/features/moderation.md

```diff
@@ -0,0 +1,18 @@
+# Administration panel
+
+The administration panel is a feature in InvenioRDM introduced in v10 that provides a graphical user interface for managing users and records. It is designed to be used by administrators and superusers to moderate the content of the repository.
+For more technical details you can read the [developer guide to the InvenioRDM administration panel](../develop/topics/administration_panel.md), detailing its programmatic interface and usage.
+
+## Moderation of users and records
+
+![The User Management](imgs/user-management.png){ loading=lazy }
+
+_Introduced in InvenioRDM v12_
+
+The administration panel now includes a "User management" section to deactivate, block and delete users, as well as undo all those actions. Deactivating a user temporarily prevents them from creating records, but lets them appeal their deactivation. Blocked users cannot sign in anymore and deleted users are removed from the system.
```

Comment:
I think "deleted users are removed from the system" should be "their records are removed from public view and replaced with a tombstone page indicating that the user has been blocked"